howtos/check_conan_version_in_conanfile.rst

```diff
@@ -11,8 +11,7 @@ Let's have a look at a basic example of this:
 .. code-block:: python
    :caption: conanfile.py
 
-    from conans import ConanFile, CMake, __version__ as conan_version
-    from conans.model.version import Version
+    from conans.tools import Version
```

Comment:
I don't think this is correct. The code still uses ``conan_version``, ``CMake``, ``ConanFile``, those imports cannot be removed